{
    "type": "bundle",
    "id": "bundle--a8f0cd0d-8744-4b7f-91fd-c3d70204f752",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--7bbb6ff5-7fc2-4dd0-aec8-ef249cd74f8e",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2023-06-08T00:05:37.950314Z",
            "modified": "2023-06-08T00:05:37.950314Z",
            "name": "CVE-2023-33595",
            "description": "CPython v3.12.0 alpha 7 was discovered to contain a heap use-after-free via the function ascii_decode at /Objects/unicodeobject.c.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2023-33595"
                }
            ]
        }
    ]
}